src/index.js

```diff
@@ -1,63 +1,84 @@
-    completed: 'false',
-    index: '2',
+    description: 'Take a shower',
+    completed: false,
+    index: 1,
   },
 ];
 
-class Tasks {
-  constructor() {
-    this.elements = JSON.parse(localStorage.getItem('list')) || list;
-  }
+const showList = () => {
+  const father = document.getElementById('allmylist');
+  father.innerHTML = '';
+  const ul = document.createElement('ul');
+  ul.classList.add('ul');
+  for (let x = 0; x < tasks.length; x += 1) {
+    if (tasks[x].completed === false) {
+      const div = document.createElement('div');
+      div.classList.add('Groupdiv');
+      div.setAttribute('id', `id${x}`);
+      const minorDiv = document.createElement('div');
+      minorDiv.classList.add('div');
+      const box = document.createElement('input');
+      box.setAttribute('type', 'checkbox');
 
-  showMyList() {
-    const theLists = document.querySelector('.allthelist');
-    theLists.innerHTML = '';
+      const li = document.createElement('li');
+      li.classList.add('li');
 
-    this.elements.forEach((item) => {
-      const full = document.createElement('div');
-      const box = document.createElement('input');
-      box.type = 'checkbox';
-      const work = document.createElement('span');
+      li.textContent = tasks[x].description;
       const points = document.createElement('img');
-      full.classList.add('grouptasks');
-      box.classList.add('completed');
-      work.classList.add('task');
-      points.classList.add('points');
-      work.textContent = item.description;
+      points.classList.add('threepoints');
+
       points.src = threepoints;
-      document.querySelector('.imagesync').src = sync;
-      document.querySelector('.push').src = push;
-      full.appendChild(box);
-      full.appendChild(work);
-      full.appendChild(points);
-      theLists.appendChild(full);
-    });
+
+      points.addEventListener('click', () => {
+        points.style.display = 'none';
+        const trash = document.createElement('img');
+        trash.src = imagetrash;
+        trash.classList.add('mytrash');
+
+        div.appendChild(trash);
+
+        trash.addEventListener('click', () => {
+          document.getElementById(`id${x}`).remove();
+          trash.style.display = 'none';
+        });
+      });
+      minorDiv.appendChild(box);
+      minorDiv.appendChild(li);
+      div.appendChild(minorDiv);
+      div.appendChild(points);
+      ul.appendChild(div);
+    }
   }
-}
+  father.appendChild(ul);
+};
 
-document.addEventListener('DOMContentLoaded', () => {
-  const organization = new Tasks();
-  organization.showMyList();
+document.addEventListener('DOMContentLoaded', showList);
+
+const input = document.getElementById('input');
+input.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    const myI = {
+      description: input.value,
+      completed: false,
+      index: tasks.length,
+    };
+    tasks.push(myI);
+    input.value = '';
+    showList();
+  }
 });
```

Comment:
- [ ] Good job so far in displaying added todos to the screen in a fly as they are added. To meet the project requirement, you're expected to save all todos added to localStorage and the todos rendered to the screen should be fetched from localStorage. Kindly work on that and take the points below into consideration:

1. By default, the new Todo should have the property `completed` set to `false` and the property `index` set to the value of the new array length (i.e. if a 5th todo is added to the list, the index of that todo should equal to 5 and not 4 as counted in array format). That can be achieved by setting the index to `index: tasks.length + 1` like this 👍🏽
1. Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. 👍🏽
1. All changes to the To-Do List should be saved in localStorage 👍🏽
2. When a task is deleted, it should be gone in the localStorage and at the same time on the screen 👍🏽